src/Permissions/Driver/Config.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaravelDoctrine\ACL\Permissions\Driver;
+
+use Illuminate\Support\Collection;
+
+class Config implements PermissionDriver
+{
+    protected Collection $collection;
+
+    /** @var array<string> */
+    public function __construct(protected array $permissions)
```

Comment:
Is defining $permissions as a protected property necessary?  It seems converting it to a collection right away and having that property, `$collection` may be all that is necessary.